ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -29,6 +30,6 @@
     version: "{{ version }}"
 
 - name: install the catalog from the catalog location
-  shell: ./installCatalog.sh {{ catalog_auth_key }} {{ api_host }} {{ catalog_namespace }} {{ cli.path }} chdir="{{ catalog_location }}/packages"
+  shell: ./installCatalog.sh {{ catalog_auth_key }} {{ api_host }} {{ catalog_namespace }} {{ cli.path }} {{ whisk_api_host }} chdir="{{ catalog_location }}/packages"
```

Comment:
instead use `whisk_api_host_name` and not create a new one, even if it's local to this file. change would be to:
```
shell: ./installCatalog.sh {{ catalog_auth_key }} {{ api_host }} {{ catalog_namespace }} {{ cli.path }} {{ whisk_api_host_name | groups['edge'] | first }} chdir="{{ catalog_location }}/packages"
```